### PR TITLE
Migrate APIs.guru related URLs

### PR DESCRIPTION
**Pull request by IvanGoncharov**
Hi @sergey-tihon 

We are migrating our URLs to `APIs.guru` domain, here is more details:
https://github.com/APIs-guru/api-models/issues/85

### CODE REVIEW

File: tests/SwaggerProvider.Tests/Schema.Parser.Tests.fs [hunk 1]
```diff
@@ -62,7 +62,7 @@ let ``Schema parse of PetStore.Swagger.json sample (online)`` () =
 
 // Test that provider can parse real-word Swagger 2.0 schemas
 // https://github.com/APIs-guru/api-models/blob/master/API.md
-type ApisGuru = FSharp.Data.JsonProvider<"http://apis-guru.github.io/api-models/api/v1/list.json">
+type ApisGuru = FSharp.Data.JsonProvider<"http://api.apis.guru/v2/list.json">
```
Comment (sergey-tihon) [inline tests/SwaggerProvider.Tests/Schema.Parser.Tests.fs]: @IvanGoncharov  Could you plz change url to `httpS://` here? Something strange happened on `Mono`
Thanks

Comment (IvanGoncharov) [inline tests/SwaggerProvider.Tests/Schema.Parser.Tests.fs]: @sergey-tihon 

> Could you plz change url to httpS:// here?

Done

> Something strange happened on Mono

It was the problem on our side we mistakenly redirected http => https on the api.apis.guru domain.

**Comment by sergey-tihon** [comment]
ssl cert validation don't pass on Mono =(
https://travis-ci.org/fsprojects/SwaggerProvider/builds/136897572#L1527

**Comment by IvanGoncharov** [comment]
> ssl cert validation don't pass on Mono =(

@sergey-tihon That's bad.
In the old API we used GitHub certificate in the new one we use one from CloudFlare.
We will try to figure out some solution ASAP

**Comment by IvanGoncharov** [comment]
@sergey-tihon I will run few experiments to see if migrating to AWS S3 will solve your problems.

**Comment by IvanGoncharov** [comment]
@sergey-tihon Looks like S3 solved all problems.
Need to run few more experiments after I setup CI deploy and connect CloudFront.

**Comment by sergey-tihon** [comment]
@IvanGoncharov Ok, just tell what a final option and when it will be ready for merge

**Comment by IvanGoncharov** [comment]
@sergey-tihon We are ready to switch to the new backend(S3 + CloudFront).
But before doing that we want to be sure that it fixed regression from API v1.
It would be great if you can workaround test somehow or as alternative run them locally.

**Comment by sergey-tihon** [comment]
Should be OK now after rebase https://github.com/fsprojects/SwaggerProvider/pull/32

**Comment by IvanGoncharov** [comment]
🎆 🎆 🎆 
It works without modification.
That mean we could safely switch to the new backend.

**Comment by IvanGoncharov** [comment]
@sergey-tihon We switched backend. 
This PR is ready for merge.

Thank you for help and patience.

P.S. Our projects use each other as testsuite :)

**Comment by sergey-tihon** [comment]
Thank you @IvanGoncharov

**Pull request merged**